### PR TITLE
Refactored Code to ES6 syntax and module

### PR DESCRIPTION
**Pull request by Asiimwe-Mark**
Used NPM and refactored some of my code to modules and introduced arrow functions and ES6 syntax.
I also used luxon to integrate a working clock into my project.

Here are the project requirements.

Project requirements
Setup your project:

Create a new directory and initialize it with git.
Use npm init -y command to create package.json file.
Create the entry point for your JavaScript code called index.js
Create index.css file for your styles.
Create the main index.html file. Link your index.js and index.css files there.
Create a directory called modules.
Re-create your Awesome books app following the rules:

Use ES6 syntax.
Divide your code into modules.
Create a separate file for each module in the modules directory.
Import modules into the index.js file. (Tip: the more modules the better!)
Keep all funcionalities of the app without errors.
Refactor your functions to arrow functions.
Use let and const in the correct way.
Display the current date below the navigation bar.

For this requirement you should use a library [Luxon](https://moment.github.io/luxon/#/install?id=es6)
Download the library into your project using npm.
Import it as a module.
See the article [Get the current date and time](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time). Your task here is to use a new tool effectively with little externally curated support. Exactly as you will do in your future job.

### CODE REVIEW

File: src/js/index.js [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+  static removeBook(id) {
+    const index = books.findIndex(book => book.id === id);
+    if (index !== -1) {
+      books.splice(index, 1);
+      booksList.innerHTML = '';
+      books.forEach(book => UI.createBook(book));
+      addBook();
+    }
+  }
+}
```
Comment (DammyShittu) [inline src/js/index.js:53]: - [ ] Part of the guidelines is that we divide the code into reasonable modules. Putting that into account, I would suggest you move this class definition containing the functionalities to another file under the modules folder. This `index.js` file should mostly be for importing classes and methods and running the app.

**Review by DammyShittu — CHANGES_REQUESTED** [review]
Hi @As1imwe-Mark,

## Highlights
✔️ Arrow functions

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/js/index.js (hunk 1 above)
Comment (Asiimwe-Mark) [reply src/js/index.js:53]: @DammyShittu, I tried every possible way to put that class in another folder but it has a lot of variables in the static functions that stay undeclared and end up throwing errors

**Comment by Asiimwe-Mark** [comment]
@DammyShittu, I tried every possible way to put that class in another folder but it has a lot of variables in the static functions that stay undeclared and end up throwing errors

**Review by DeliceLydia — APPROVED** [review]
## Hi @As1imwe-Mark, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 


### Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**